AppFeedback.podspec

```diff
@@ -1,24 +1,29 @@
 Pod::Spec.new do |s|
-  s.name         = "AppFeedback"
-  s.version      = "1.0.0"
-  s.summary      = "You can post feedback messages and screenshots to Slack from your app!"
+  s.name                  = "AppFeedback"
+  s.version               = "1.0.1"
+  s.summary               = "You can post feedback messages and screenshots to Slack from your app!"
 
-  s.description  = <<-DESC
-                   Feature
-                   - Post feedback message to Slack
-                   - Show floating button for feedback
-                   - Two fingers long press to show feedback dialog
-                   - Take a screenshot & Record screen
-                   DESC
+  s.description           = <<-DESC
+                            Feature
+                            - Post feedback message to Slack
+                            - Show floating button for feedback
+                            - Two fingers long press to show feedback dialog
+                            - Take a screenshot & Record screen
+                            DESC
 
-  s.homepage     = "https://github.com/yahoojapan/AppFeedback.git"
-  s.license      = "MIT"
-  s.author       = "Yahoo Japan Corporation"
+  s.homepage              = "https://github.com/yahoojapan/AppFeedback.git"
+  s.license               = "MIT"
+  s.author                = "Yahoo Japan Corporation"
 
-  s.ios.deployment_target = "8.0"
+  s.ios.deployment_target = "9.0"
+  s.platform = :ios, '9.0'
```

Comment:
```suggestion
```

Sorry it was my misunderstanding. It was no problem if `ios.deployment_target` was specified.

Reply:
( Please correct it again if there is a problem. )